adr/0017-hardware-screening-os.md

```diff
@@ -0,0 +1,34 @@
+# 0017. Hardware screening for OS
+
+Date: 2020-10-23
+
+## Status
+
+Draft
+
+## Context
+
+We need take a decision for the minimal requirements of a IoT-Board to become a Home Assistant OS image.
+Also how we deal with old or replaced generation/boards.
+
+## Decision
+
+Follow are the requirements of a Board:
+
+- The board need to be available for everbody
+- The manifacture produce and sell the board at least 3 years
+- The manifacture is know for his continuity
+- The board need to be supported by mainstream Linux kernel with current or next LT
```

Comment:
If it is supported by the upstream Kernel, it will come into the next long term anyways...

```suggestion
- The board needs to be supported by the upstream Linux kernel
```

Reply:
Btw, maybe we need also mention what features should be supported at least. E.g. Ethernet, USB and SD-card...